QUICKSTART.md

```diff
@@ -225,6 +225,36 @@ public void handle(HttpExchange he) {
 
 ## Metrics
 
+Spans are a great way to get detailed information about what your application is doing, but
+what about a more aggregated perspective? OpenTelemetry provides supports for metrics, a time series
+of numbers that might express things such as CPU utilization, request count for an HTTP server, or a
+business metric such as transactions.
+
+All metrics can be annotated with labels: additional qualifiers that help describe what
+subdivision of the measurements the metric represents.
+
+The following is an example of metric usage:
+
+```java
+// Gets or creates a named meter instance
+Meter meter = OpenTelemetry.getMeterProvider().get("instrumentation-library-name","semver:1.0.0");
```

Comment:
I'd remove the "semver:" from the beginning of the version.

Reply:
That is in the version semantic convention